cmd/controller/main.go

```diff
@@ -34,6 +35,8 @@ import (
 	"github.com/kubermatic/machine-controller/pkg/machines"
 	"github.com/kubermatic/machine-controller/pkg/signals"
 	"github.com/kubermatic/machine-controller/pkg/ssh"
+	"github.com/oklog/run"
+	"github.com/pkg/errors"
```

Comment:
why are we introducing this library?
Whats the benefit of this compared against a simple `fmt.Errorf` ?